requirements/common.in

```diff
@@ -79,7 +79,7 @@ httplib2==0.12.1
 hypchat==0.21
 
 # For doing highly usable Python profiling
-line_profiler==2.1.2
+-e git+https://github.com/zulip/line_profiler.git#egg=line_profiler==2.1.2.zulip1
```

Comment:
@timabbott 
I have a couple of questions here,
Python 3.7 breaks the line_profiler package on pypi since it uses a precompiled '.c' file which was compiled using an older version of Cython. The latest commits you merged in zulip/line_profiler fixes these issues by compiling them at build time by specifying Cython as a requirement in pyproject.toml (PEP 518). But since pip 19.1, editable mode is not supported for projects with build-backend (PEP 517) defined in their pyproject.toml file. And pip-compile doesnt support URLs as packages unless they are editable. Also, the --no-use-pep517 option is not yet supported inside requirements files(There is currently a PR for that).
One workaround for this situation would be to compile the pyx files with the latest version of cython and push it to zulip/line_profiler so it would work with python 3.7 without needing to compile at runtime.
I thought I had tested it properly when I asked you to merge the PR to zulip/line_profiler but I made a mistake somewhere for which I'm really sorry for.

Reply:
Not a big deal; can you try submitting a PR to `zulip/line_profiler` with the Python 3.7 PXC build?

Reply:
Just submitted a PR to zulip/line_profiler for this

Reply:
@timabbott The failing tests should pass once this [PR](https://github.com/zulip/line_profiler/pull/1) is merged.

Reply:
@mayankmadan merged.

Reply:
It should be good to go now. I'll create a topic for this in "code review" so someone else could test it out.